frog/responsive-images.rkt

```diff
@@ -0,0 +1,117 @@
+#lang racket/base
+
+(require net/url
+         racket/contract/base
+         racket/contract/region
+         racket/file
+         racket/function
+         racket/list
+         racket/path
+         racket/port
+         racket/system
+         racket/string
+         "params.rkt"
+         "util.rkt"
+         "verbosity.rkt"
+         (only-in srfi/1 list-index))
+
+(provide make-responsive clean-resized-images magick-available?)
+
+(define magick-available? (find-executable-path "magick"))
+
+(module+ test
+  (require rackunit))
+
+(define (image-width path)
```

Comment:
Could this be simplified? I'm not sure I understand why there's the extra `magic-width` function, much less defining `get-width` and calling that? Couldn't the definition of `image-width` simply/directly be what is currently the body of `magic-width`?